src/main/resources/jnt_template/html/template.content-manager.jsp

```diff
@@ -34,7 +34,7 @@
         <script type="text/javascript">
             window.top.DX && window.top.DX.switch("contentmanager", {chrome: false});
             contextJsParameters['siteKey'] = '${renderContext.mainResource.node.resolveSite.name}';
-            contextJsParameters['siteDisplayableName'] = '${renderContext.mainResource.node.resolveSite.displayableName}';
+            contextJsParameters['siteDisplayableName'] = '${functions:escapeJavaScript(renderContext.mainResource.node.resolveSite.displayableName)}';
             contextJsParameters['urlBrowser'] = '/cms/contentmanager';
             contextJsParameters['urlbase'] = '${renderContext.servletPath}';
             contextJsParameters['langName'] = '${functions:displayLocaleNameWith(mainResourceLocale, mainResourceLocale)}';
```

Comment:
Maybe you can escape the username too ?

Reply:
yep could be useful, I'll do a second commit